packages/components-css/icon-css/src/_mixin.scss

```
@@ -1,21 +1,46 @@
 @mixin nl-icon {
-  block-size: var(--nl-icon-block-size, 1em);
+  // Default to 1lh × 1lh
+  --_nl-icon-block-size: 1lh;
+  --_nl-icon-inline-size: 1lh;
```

```suggestion
  --_nl-icon-block-size: 1ic;
  --_nl-icon-inline-size: 1ic;
```

Als iemand een grotere line-height instelt, wil je niet per se een grotere icon. De `ic` unit lijkt eigenlijk wel heel geschikt voor onze doeleinden.

---

> `ic`
> Equal to the used advance measure of the "水" glyph (CJK water ideograph, U+6C34), found in the font used to render it.

Ik ben wel benieuwd of `ic` problemen oplevert met fonts waarin dit karakter niet is ingesteld...